CalculatorOperations.UnitTests/RomanTests.cs

```diff
@@ -14,12 +14,14 @@ public void CheckResultParseRomanToAradNumber(string roman, int arab)
         }
 
         [Theory]
-        [InlineData("IV", "4")]
-        public void ShowRomanToAradNumber(string roman, string expected)
+        [InlineData("V", "V")]
+        public void ShowRomanToAradNumber(string roman, string expectedResult)
         {
+            string res = string.Empty;
             var parsedRomanNumeric = RomanNumericsNumber.Parse(roman, null);
-            var parsedNumberToString = parsedRomanNumeric.ToString();
-            Assert.Equal(expected, parsedNumberToString);
+            parsedRomanNumeric.NumberRoman = res;
```

Comment:
not sure this line.Does it correct?I don't know where I must to call this property.If doesn't call this prop-it"s null.